week4/assignment/src/pages/mypage/mypage.jsx

```diff
@@ -0,0 +1,94 @@
+import { Profile } from "../../assets/DATA";
+import { Wrapper, Title } from "../../styles/CommonStyle";
+import PerInput from "../../components/common/input";
+import styled from "styled-components";
+import { useNavigate, useParams } from "react-router-dom";
+import axios from "axios";
+import { useState, useEffect } from "react";
+
+const MyPage = () => {
+  const [getUsername, setGetUsername] = useState("");
+  const [getNickname, setGetNickname] = useState("");
+
+  const { userId } = useParams();
+
+  const navigate = useNavigate();
+
+  const handleGet = async () => {
+    try {
+      const response = await axios.get(
+        `${import.meta.env.VITE_APP_BASE_URL}/api/v1/members/${userId}`,
+        {
+          userId: userId,
+        }
+      );
+      const { data } = response;
+      setGetUsername(data.username);
+      setGetNickname(data.nickname);
+    } catch {
+      console.log("error");
+    }
```

Comment:
이렇게 하면 무슨 error인지 더 알기 쉽겠징?!
```suggestion
    catch (error) {
        console.error(error);
      }
```